packages/vkui/src/components/Group/Group.tsx

```diff
@@ -111,7 +107,9 @@ export const Group = ({
         {header}
         {children}
         {hasReactNode(description) && (
-          <Footnote className={styles['Group__description']}>{description}</Footnote>
+          <Footer Component="span" mode="group">
+            {description}
+          </Footer>
```

Comment:
```suggestion
          <Footer mode="group">{description}</Footer>
```

Предлагаю тут оставить семантически верный `footer`.

Reply:
Зря быканула (2), резолвлю в пользу отдельного issue: 
- https://github.com/VKCOM/VKUI/issues/4692